Project-1/README.md

```diff
@@ -67,9 +183,99 @@ Note: The standard interpretation of the logical symbols - "∨", "∧", "→",
   (b) ∀x∀y(¬(Px ∨ Qx) → (¬Px ∧ ¬Qx))
   (c) ∀x∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
   (d) ∀x∀y((¬Px ∧ ¬Qx) → ¬(Px ∨ Qx))
+  
+   (a) ∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+  
+ (1) ¬∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+ (2) ¬∀y(¬(Pa ∧ Qa) → (¬Pa ∨ ¬Qa))
+ (3) ¬(¬(Pa ∧ Qa) → (¬Pa ∨ ¬Qa))
+ (4)¬(Pa ∧ Qa)
+ (5)¬(¬Pa ∨ ¬Qa)
+ (6).¬¬Pa
+ (7)¬¬Qa
+ (8)Qa
+ (9) Pa
+ (10) ¬Pa	(11) ¬Qa
+        x	       x
+ 
+  (b) ∀x∀y(¬(Px ∨ Qx) → (¬Px ∧ ¬Qx))
+  
+  (1) ¬∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+  (2) ¬∀y(¬(Pa ∧ Qa) → (¬Pa ∨ ¬Qa))
+  (3) ¬(¬(Pa ∧ Qa) → (¬Pa ∨ ¬Qa))
+  (4) ¬(Pa ∧ Qa)
+  (5) ¬(¬Pa ∨ ¬Qa)
+  (6) ¬¬Pa
+  (7) ¬¬Qa
+  (8) Qa
+  (9) Pa
+  (10) ¬Pa	(11) ¬Qa
+        x	      x
+  
+  (c) ∀x∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
+  
+  (1) ¬∀x∀y((¬Px ∨ ¬Qx) → ¬(Px ∧ Qx))
+  (2) ¬∀y((¬Pa ∨ ¬Qa) → ¬(Pa ∧ Qa))
+  (3) ¬((¬Pa ∨ ¬Qa) → ¬(Pa ∧ Qa))
+  (4) ¬Pa ∨ ¬Qa
+  (5) ¬¬(Pa ∧ Qa)
+  (6) Pa ∧ Qa
+  (7) Pa
+  (8) Qa
+  (9) ¬Pa	(10) ¬Qa
+     x		     x
+  
+  (d) ∀x∀y((¬Px ∧ ¬Qx) → ¬(Px ∨ Qx))
+  
+ (1) ¬∀x∀y((¬Px ∧ ¬Qx) → ¬(Px ∨ Qx))
+ (2) ¬∀y((¬Pa ∧ ¬Qa) → ¬(Pa ∨ Qa))
+ (3) ¬((¬Pa ∧ ¬Qa) → ¬(Pa ∨ Qa))
+ (4) ¬Pa ∧ ¬Qa
+ (5) ¬¬(Pa ∨ Qa)
+ (6) Pa ∨ Qa
+ (7) ¬Pa
+ (8) ¬Qa
+ (9) Pa		(10) Qa
+     x		     x
+     
 ```
 	
 9. Using a natural deduction proof generator - such as the one found here `https://proofs.openlogicproject.org/` - provide natural deduction proofs for each of De Morgan's laws. 
 
+¬∀x∀y(¬(Px ∧ Qx) → (¬Px ∨ ¬Qx))
+(1) ¬(Px ∧ Qx)		assumption
+(2) ¬(¬Px ∨ ¬Qx)	assumption
+(3) (Px ∧ Qx)		2, negated disjunction
+(4) ¬(Px ∧ Qx)		1,3 contradiction
+
+∀x∀y(¬(Px ∨ Qx) → (¬Px ∧ ¬Qx))
+(1) ¬(Px ∨ Qx)		assumption
+(2) (¬Px ∧ ¬Qx)		assumption
```

Comment:
You're missing a negation, I believe

Reply:
Line two, in the second assumption should have a negation before the conjunction (I think): ¬(¬Px ∧ ¬Qx)